integration/service/src/main/java/com/cerner/beadledom/integration/service/resource/HelloWorldResource.java

```diff
@@ -0,0 +1,43 @@
+package com.cerner.beadledom.integration.service.resource;
+
+import com.cerner.beadledom.integration.api.model.HelloWorldDto;
+import com.cerner.beadledom.jaxrs.GenericResponse;
+import com.cerner.beadledom.jaxrs.GenericResponses;
+import com.cerner.beadledom.pagination.OffsetPaginatedList;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import java.util.ArrayList;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+/**
+ * Server implementation of an offset based paginated list endpoint.
+ *
+ * @author Nick Behrens
+ */
+@Api(value = "/hello", description = "Retrieve hello world data")
```

Comment:
description is deprecated

Reply:
It is, I just removed it. There didn't seem to be a viable replacement, but this is just an example service so I'm not too worried about the swagger docs being good: https://github.com/cerner/beadledom/pull/149/commits/0cfffed54c3dbbcc7b53efa136c19292644441fb